### ISSUE TITLE
Unable to play (apparently) valid module

### ISSUE THREAD

**Issue by cspiegel** [cmatsuoka/libxmp#48]
sandman.mod from [here](http://modarchive.org/index.php?request=view_by_moduleid&query=54248) is not recognized by libxmp, due to the following checks in ```src/loaders/mod_load.c```, which are done as a “[s]anity check to prevent loading NoiseRunner and other module formats with valid magic at offset 1080”:

```
if (hio_read8(f) & 0xf0)                /* test finetune */
        return -1;
if (hio_read8(f) > 0x40)                /* test volume */
        return -1;
```
If I modify the above to not return failure, the file loads and plays fine.  Apparently this module looks enough like a NoiseRunner module to trigger the tests, but it's not one.  I'm not sure if the module is partially corrupted or if the test result is a false positive.

**Comment by cmatsuoka** [comment]
I checked the module and it has an (apparently) invalid finetune value in instrument 12, with upper four bits set to 2. The format "specs" say:

      44      1    Lower four bits are the finetune value, stored as a signed
                   four bit number. The upper four bits are not used, and
                   should be set to zero.

But reality is messier than that and you can't really expect all modules to follow the spec, so I changed the test to allow sandman.mod to get away with that. Fix will available in 4.3.11 to be released later today.

**Comment by mldoering** [comment]
Am Sat, 13 Feb 2016 03:03:20 -0800
schrieb Claudio Matsuoka <notifications@github.com>:

> Closed #48.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cmatsuoka/libxmp/issues/48#event-549417951

If I recognize it correctly #12 is an empty instrument so fine tune
should be ignored anyway.

It would probl. be interesting to convert it into a Noiserunner and see
if it screws up detection.

Anyway, I think it is enough safe to play it to risk
anything.

mld/uade team

Ps. interesting mod for me, too.  Seems I have made a mistake when
calculating filesize. ;-)
